### ISSUE TITLE
Getbooks: use console.log to show actual error (instead of alert())

### ISSUE THREAD

**Issue by sladen** [ShadowItaly/clas-digital#115]
Currently `alert("Something went wrong please report this to the system admins");` is activated when something goes wrong at:

* https://github.com/ShadowItaly/clas-digital/blob/cpp/web/GetBooks.html#L784

This does not *say* what the error is that the user should "report" is, or who the "systems admins" are.  If there is an exact error this can be saved via `console.log`, or adding content to the OCR div, showing exactly what went wrong.

**TL;DR** uses of `alert()` should be removed.

**Comment by alleonhardt** [comment]
The error is removed, could not load hit list is shown instead. Fixed by commit https://github.com/ShadowItaly/clas-digital/commit/f9c64e6fb98f355d604a32a96cd8eaa7a81ccd38
